devicetypes/smartthings/zigbee-ceiling-fan-light.src/led-fan-lightings.groovy

```diff
@@ -62,15 +62,15 @@ private getON_OFF_CLUSTER_VALUE() { 0x0006 }
 def parse(String description) {
 	// Parse incoming device messages to generate events
 	log.debug "description is $description"
-	def event = zigbee.getEvent(description)
-	def zigbeeMap = zigbee.parseDescriptionAsMap(description)
+	def event = zigbee.getEvent(description)	
 	if (event) {
 		sendEvent(event)
 	} else if (description?.startsWith('read attr -')) {
+		def zigbeeMap = zigbee.parseDescriptionAsMap(description)
 		if (zigbeeMap.clusterInt == FAN_CLUSTER_VALUE &&
 		    zigbeeMap.attrInt == FAN_STATUS_VALUE) {		
 			log.debug "read attribute event for fan cluster attrib FAN_STATUS"
-			def childDevice = getChildDevices()?.find {
+			def childDevice = childDevice.find {
```

Comment:
`getChildDevices()` => `childDevices`, not `childDevice`. Plural, not singular.

Reply:
Thank you very much. By your review the codes go to accurate.